website/users/migrations/0002_auto_20200706_1400.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.0.5 on 2020-07-06 12:00
+
+import django.contrib.auth.models
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
We might want to squash the migrations in this pull request or before the first release.